metacat-client/src/main/java/com/netflix/metacat/client/api/ParentChildRelV1.java

```diff
@@ -40,4 +41,24 @@ Set<ChildInfoDto> getChildren(
         @PathParam("table-name")
         String tableName
     );
+
+    /**
+     * Return the list of parent.
+     * @param catalogName catalogName
+     * @param databaseName databaseName
+     * @param tableName tableName
+     * @return list of parent info dtos
+     */
+    @GET
+    @Path("parents/catalog/{catalog-name}/database/{database-name}/table/{table-name}")
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.APPLICATION_JSON)
+    Set<ParentInfoDto> getParents(
```

Comment:
one the one hand, i would suggest we rename this to `getParent` since we are enforcing the one-parent-per-child rule and its a little confusing as the name implies we can expect more than one results. on the other hand, this API does technically return more than one result, so we would need to have a different return type to make this change. i have a weak preference to rename it and change the contract despite it becoming less flexible, but its up to you

Reply:
If the api stays having multiple return values then plural is better so that the correspondence holds, otherwise singular. There is no current use case for multiple parents but keeping the flexibility is reasonable.